core/trino-main/src/test/java/io/trino/sql/query/TestOrderedAggregation.java

```diff
@@ -103,10 +103,6 @@ public void testAggregationWithOrderBy()
                 "SELECT x, y, array_agg(z ORDER BY z) FROM (VALUES (1, 2, 3), (1, 2, 1), (2, 1, 3), (2, 1, 4)) t(x, y, z) GROUP BY GROUPING SETS ((x), (x, y))"))
                 .matches("VALUES (1, NULL, ARRAY[1, 3]), (2, NULL, ARRAY[3, 4]), (1, 2, ARRAY[1, 3]), (2, 1, ARRAY[3, 4])");
 
-        assertThat(assertions.query(
-                "SELECT array_agg(z ORDER BY z) OVER (PARTITION BY x) FROM (VALUES (1, 2, 3), (1, 2, 1), (2, 1, 3), (2, 1, 4)) t(x, y, z) GROUP BY x, z"))
-                .failure().hasMessageMatching(".* Window function with ORDER BY is not supported");
-
```

Comment:
Leave the test but use another window function (not an aggregate).

Reply:
how does this match `TestOrderedAggregation`

I can do 
```
        assertThat(assertions.query(
                "SELECT abs(z ORDER BY z) OVER (PARTITION BY x) FROM (VALUES (1, 2, 3), (1, 2, 1), (2, 1, 3), (2, 1, 4)) t(x, y, z) GROUP BY x, z"))
                .failure().hasMessageMatching(".* ORDER BY is only valid for aggregation functions");
```

but it is kinda tested in TestAnalyzer already (to some extent)

Reply:
> how does this match TestOrderedAggregation

Good point. In this case, could we assert a successful query?

Reply:
It is tested verbosly in TestWindow already

Reply:
I think it would be worthy to have a test here for completeness but feel free to decide.

Reply:
readded